user-timing/measure_exception.html

```diff
@@ -19,16 +19,16 @@ <h1>Description</h1>
 <div id="log"></div>
 <script>
 performance.mark('ExistMark');
-test_method_throw_exception('performance.measure()', TypeError());
+test_method_throw_exception('performance.measure()', TypeError);
 test_method_throw_exception('performance.measure("Exception1", "NonExistMark1")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception2", "NonExistMark1", "navigationStart")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception3", "navigationStart", "NonExistMark1")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception4", "NonExistMark1", "ExistMark")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception5", "ExistMark", "NonExistMark1")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception6", "NonExistMark1", "NonExistMark2")', 'SYNTAX_ERR');
 test_method_throw_exception('performance.measure("Exception7", "redirectStart")', 'INVALID_ACCESS_ERR');
-test_method_throw_exception('performance.measure("Exception8", {"detail": "non-empty"})', TypeError());
-test_method_throw_exception('performance.measure("Exception9", {"start": 1, "duration": 2, "end": 3})', TypeError());
+test_method_throw_exception('performance.measure("Exception8", {"detail": "non-empty"})', TypeError);
+  test_method_throw_exception('performance.measure("Exception9", {"start": 1, "duration": 2, "end": 3})', TypeError);
```

Comment:
nit: Why the extra space here?

Reply:
Good question.  Not sure how that snuck in!  Fixed.